### PR TITLE
Updated CFEExchangeCalendar to include adhoc holidays

### PR DESCRIPTION
**Pull request by trical09**
HurricaneSandyClosings and USNationalDaysofMourning added

@FreddieV4 
#1682

### CODE REVIEW

**Comment by FreddieV4** [comment]
Hey @trical09 thanks for the PR! It looks like the CI check didn't pass though. After looking at the trace, it looks like you'll need `from itertools import chain` at the top, if you want to call the `chain` function

File: zipline/utils/calendars/exchange_calendar_cfe.py [hunk 1]
```diff
@@ -62,6 +64,13 @@ def regular_holidays(self):
         ])
 
     @property
+    def adhoc_holidays(self):
+        return list(chain(
+            HurricaneSandyClosings,
+            USNationalDaysofMourning,
+            ))
```
Comment (FreddieV4) [inline zipline/utils/calendars/exchange_calendar_cfe.py]: Can you line this up with the start of the `return` statement?

**Comment by trical09** [comment]
Thanks @FreddieV4 - noob mistake. Really appreciate. Fixed, let's see how the checks go.

**Comment by FreddieV4** [comment]
Hey @trical09 i think the implementation you have here is fine:

```
def adhoc_holidays(self):
    return list(chain(
        HurricaneSandyClosings,
        USNationalDaysofMourning,
        )
    )
```

Did you mean to close this PR?

**Pull request closed without merging**